prepare-install.js

```diff
@@ -8,9 +8,6 @@ const isNode12 = process.version.startsWith('v12.');
 
 if (isWin || isNode12) {
```

Comment:
I think you can remove the isNode12 check here and only delete yarn.lock for windows

Reply:
Added.